assets/js/components/ExecutionResults/checksUtils.test.js

```diff
@@ -49,13 +51,50 @@ describe('checksUtils', () => {
     });
   });
 
+  it('getCatalogCategoryList should return a sorted category list where it is matching', () => {
+    const fakerListID = [
+      faker.datatype.number(),
+      faker.datatype.number(),
+      faker.datatype.number(),
+    ];
+    const execution = [
```

Comment:
`execution` might be misleading; what we provide to `getCatalogCategoryList`, besides the catalog, is a list of `checksResults`. Better be consistent with naming.